### PR TITLE
Add item class fields

### PR DESCRIPTION
**Pull request by KareemSabryKhaled**

### CODE REVIEW

File: Models/Item.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+class item(){
+   var id;
```
Comment (Nufall) [inline Models/Item.php]: $id; instead of var id;

File: Models/Item.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+   string name;
+   string color;
+   var quantity;
+   var itemTypeId;
+   var price;
```
Comment (Nufall) [inline Models/Item.php]: change each type to $ + attribute

**Review by Nufall — CHANGES_REQUESTED** [review]
fix your code based on comments please

File: Models/Item.php (hunk 2 above)
Comment (KareemSabryKhaled) [reply Models/Item.php]: fixed the mistakes

File: Models/Item.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?php
+class item(){
+   $id;
+   $name;
```
Comment (Nufall) [inline Models/Item.php]: set and get for each one

**Review by Nufall — CHANGES_REQUESTED** [review]
please add setters and getters for all properties

File: Models/Item.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<?php
+class item(){
+   protected $id;
+   protected $name;
+   protected $color;
+   protected $quantity;
+   protected $itemTypeId;
+   protected $price;
+
```
Comment (Nufall) [inline Models/Item.php]: where is your constructor?

**Review by Nufall — COMMENTED** [review]
please add a constructor

**Pull request merged**